### PR TITLE
[t211]

### PR DESCRIPTION
**Pull request by secretrobotron**
getHTML functionality (must use console for now)

### CODE REVIEW

**Comment by jbuck** [comment]
For HTML validation, can we s/butter-exclude/data-butter-exclude/ ?

File: src/butter.js [hunk 1]
```diff
@@ -40,11 +40,11 @@
         path = path.join( '/' ) + '/';
 
         if ( !window.require ) {
-          document.write( '<script src="' + path + '../external/require/require.js"></' + 'script>' );
+          document.write( '<script butter-exclude="true" src="' + path + '../external/require/require.js"></' + 'script>' );
```
Comment (jbuck) [inline src/butter.js]: should it be butter-exclude=true or just butter-exclude ?

File: src/core/page.js [hunk 2]
```diff
@@ -40,6 +40,48 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
...
+        var node = toExclude[ i ];
+        node.parentNode.removeChild( node );
+      } //for
+
+      toClean = body.querySelectorAll( "*[butter-clean=\"true\"]" );
+      for( var i=0, l=toClean.length; i<l; ++i ){
+        var node = toClean[ i ];
+        node.removeAttribute( "butter-clean" );
+        node.removeAttribute( "data-butter" );
+        node.className = node.className.replace( /ui-droppable/g, "" );
```
Comment (jbuck) [inline src/core/page.js]: Using a regexp to delete text? that's AWFUL! But classList (https://developer.mozilla.org/en/DOM/element.classList) doesn't exist everywhere, so bleh

File: src/butter.js (hunk 1 above)
Comment (secretrobotron) [inline src/butter.js]: I don't know if it matters. Like `controls` in a `video` tag. I never put `="true"`. Is that a HTML validation infringement too?

Comment (secretrobotron) [inline src/butter.js]: The w3c page on <video> seems to not care. I prefer to leave it off because it's prettier and harder to screw up.

Comment (jbuck) [inline src/butter.js]: That's cool, I just wanted consistency. and data-butter-exclude is shorter!

**Pull request merged**